README.md

```diff
@@ -1,73 +1,84 @@
 
-HE has endeavored to prevent the population of these states; for that purpose obstructing the laws for naturalization of foreigners, refusing to pass others to encourage their migrations hither, & raising the conditions of new appropriations of lands.
+He has endeavored to prevent the population of these states; for that purpose obstructing the laws for naturalization of foreigners, refusing to pass others to encourage their migrations hither, & raising the conditions of new appropriations of lands.
 
-HE has suffered the administration of justice totally to cease in some of these states refusing his assent to laws for establishing judiciary powers.
+He has obstructed the administration of justice by refusing his assent to laws for establishing judiciary powers.
 
-HE has made our judges dependant on his will alone, for the tenure of their offices, & the amount & paiment of their salaries.
+He has made our judges dependant on his will alone, for the tenure of their offices, & the amount & paiment of their salaries.
 
-HE has erected a multitude of new offices by a self assumed power and sent hither swarms of new officers to harass our people and eat out their substance.
+He has erected a multitude of new offices, and sent hither swarms of officers to harass our people and eat out their substance.
 
-HE has kept among us in times of peace standing armies and ships of war without the consent of our legislatures.
+He has kept among us, in times of peace, standing armies without the consent of our legislatures.
 
-HE has affected to render the military independent of, & superior to the civil power.
+He has affected to render the military independent of, & superior to the civil power.
 
-HE has combined with others to subject us to a jurisdiction foreign to our constitutions & unacknowledged by our laws, giving his assent to their acts of pretended legislation:
+He has combined with others to subject us to a jurisdiction foreign to our constitutions & unacknowledged by our laws, giving his assent to their acts of pretended legislation:
 
-FOR quartering large bodies of armed troops among us:
+For quartering large bodies of armed troops among us:
 
-FOR protecting them by a mock-trial from punishment for any murders which they should commit on the inhabitants of these states
+For protecting them by a mock-trial from punishment for any murders which they should commit on the inhabitants of these states:
 
-FOR cutting off our trade with all parts of the world:
+For cutting off our trade with all parts of the world:
 
-FOR imposing taxes on us without our consent:
+For imposing taxes on us without our consent:
 
-FOR depriving us of the benefits of trial by jury
+For depriving us in many cases, of the benefit of Trial by Jury:
```

Comment:
These injuries are not vestigial or imaginary, but incalculable sins perpetrated upon the citizens of this great Nation as we speak.